drivers/AnalogOut.h

```diff
@@ -138,7 +138,10 @@ class AnalogOut {
 
     virtual ~AnalogOut()
     {
-        // Do nothing
+        /** Deinitialize the pin configuration totally. 
+         *\ This should allow changing the definition of the pin
+         */
+          analogout_free(&this->_dac);
```

Comment:
```suggestion
         analogout_free(&_dac);
```
Private members are already marked with a `_` prefix; no need to use `this` pointer here.